r_basics_introduction/r_basics_introduction.md

```diff
@@ -775,6 +775,7 @@ RStudio's Data Import cheat sheet helps with importing data from file types simi
 *R for Data Science* is a free text that provides lots of helpful explanation and examples.  The [section on data import](https://r4ds.had.co.nz/data-import.html) goes into much more detail than we were able to do in this brief module.
 
 The entire text is available [in English](https://r4ds.had.co.nz/) and [in Spanish](https://es.r4ds.hadley.nz/).
+There's also an [unofficial solutions guide](https://jrnold.github.io/r4ds-exercise-solutions/index.html) (only available in English) to allow you to check your work. 
 </div>
 <div style = "margin: 1rem; max-width: 20%; float:left;">
 ![Book Cover](media/r4ds.png)<!-- style = "max-width: 90%" -->
```

Comment:
```suggestion
![""](media/r4ds.png)<!-- style = "max-width: 90%" -->
```

Reply:
@pm0kjp just this one final little change and we should be good to merge it in! 😄